waveform-django/waveforms/views.py

```diff
@@ -124,6 +129,60 @@ def admin_console(request):
     # Get all the current users
     all_users = User.objects.all()
 
+    # Allow admins to accept annotation requests
+
+    if request.method == 'POST':
+        if 'give_annotations' in request.POST:
+            annotator = User.objects.get(username=request.POST['give_annotations'])
+
+            # Get CSV data
+            csv_fields = ['Record', 'Users Assigned']
+            csv_rows = {}
+            with open(csv_path, 'r') as csv_file:
+                csvreader = csv.reader(csv_file, delimiter=',')
+                next(csvreader)
+                for row in csvreader:
+                    names = []
+                    for val in row[1:]:
+                        if val:
+                            names.append(val)
+                    csv_rows[row[0]] = names
+
+            # Add records to CSV if not already present
+            for rec in all_records:
+                if rec not in csv_rows.keys():
+                    csv_rows[rec] = []
+
+            # Add records to User
+            recs_to_ann = 20
```

Comment:
Do you think these should be global variables in the settings file so we can adjust them easily?